curiosrc/seal/task_treed.go

```diff
@@ -178,82 +239,25 @@ func (t *TreesTask) Do(taskID harmonytask.TaskID, stillOwned func() bool) (done
 		unpaddedData = false // nullreader includes fr32 zero bits
 	}
 
-	sref := storiface.SectorRef{
-		ID: abi.SectorID{
-			Miner:  abi.ActorID(sectorParams.SpID),
-			Number: abi.SectorNumber(sectorParams.SectorNumber),
-		},
-		ProofType: sectorParams.RegSealProof,
-	}
-
-	// D / R / C
-	sealed, unsealed, err := t.sc.TreeDRC(ctx, &taskID, sref, commd, abi.PaddedPieceSize(ssize), dataReader, unpaddedData)
+	// Generate Tree D
+	err = t.sc.TreeD(ctx, sref, commd, abi.PaddedPieceSize(ssize), dataReader, unpaddedData, fsPaths, pathIds, release)
```

Comment:
If we defer-call release here we shouldn't call it in TreeD, that's a double call, and that's bad (tho at least now guarded against, so no big consequences)

Reply:
I remember removing the extra defer. I guess I lost the local change somehow before the push. 
Removed now.